src/menu.rs

```diff
@@ -4,7 +4,11 @@
 use music;
 use opengl_graphics::GlGraphics;
 use opengl_graphics::glyph_cache::GlyphCache;
-use piston_window::{Button, clear, Context, Input, Key, PistonWindow, text, Transformed, Size};
+use piston::event_loop::Events;
```

Comment:
In general, I do think using piston core makes it a little more explicit where all these piston types are being imported from, so I think that's a plus.